### PR TITLE
Ft user login #182201678

### PR DESCRIPTION
**Pull request by alaintresor**
#### What does this PR do?
User login frontend

#### How should this be manually tested?

- Clone the repo.
- Run npm install to install modules.
- Run npm start to run an app.
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/182201678
#### Screenshots 
Screenshot (49)

### CODE REVIEW

File: src/views/Login.js [hunk 1]
```diff
@@ -1,10 +1,107 @@
...
+          localStorage.setItem('token', token);
+          localStorage.setItem('loggedUser', JSON.stringify(user));
+          dispatch(logginUser(user));
+          dispatch(loginMode());
+          swal.fire(`Hey ${user.username}`, 'Welcome to Barefoot Nomad', 'success');
+        // props.history.push('/');
+        })
+        .catch((err) => {
+          console.log(err);
+          swal.fire('Oops...', `${err.response.data.message}`, 'error');
```
Comment (joshwambere) [inline src/views/Login.js:45]: may be redirect to other page instead of firing this modal

File: src/views/Login.js [hunk 2]
```diff
@@ -1,10 +1,107 @@
...
+                        </div>
+                        <div className="form-group">
+                            <div className="input-group" id="password">
+                                <input type="password" name="" placeholder="Password" value={password} onChange={getPassword}/>
+                                <ion-icon name="lock-closed-outline"></ion-icon>
+                            </div>
+                        <small id="password-error" className="error"></small>
+                        </div>
+                        <div className="forgot-p">
+                            <p>Forgot password?</p>
```
Comment (joshwambere) [inline src/views/Login.js:78]: increase this text fontSize

File: src/redux/store.js [hunk 3]
```diff
@@ -1,8 +1,18 @@
 import { configureStore } from "@reduxjs/toolkit";
 import counterReducer from "./counter/counterSlice";
+import isLoggedReducer from "./auth/isLoggedSlice";
+import loginReducer from "./auth/loginSlice";
+
+// eslint-disable-next-line import/prefer-default-export
```
Comment (joshwambere) [inline src/redux/store.js]: remove this comment

File: src/tests/mocks/handlers/auth.handler.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  rest.post(`https://bravo-bfn-be.herokuapp.com/api/v1/user/login`, (req, res, ctx) => {
+    const { email } = req.body;
+    if (email.includes('wrong')) {
+      return res(
+        ctx.status(400),
+        ctx.json({ Error: 'wrong email or password', status: 400 }),
+        ctx.delay(100)
+      );
+    }
+    console.log("doneeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee")
```
Comment (joshwambere) [inline src/tests/mocks/handlers/auth.handler.js]: remove this console

File: src/views/Login.js [hunk 5]
```diff
@@ -1,10 +1,107 @@
...
 
-    );
-}
\ No newline at end of file
+  // eslint-disable-next-line consistent-return
+  const submitHandle = (e) => {
+    e.preventDefault();
+
+    if (loginValidation(email, password)) {
+      axios.post('https://bravo-bfn-be.herokuapp.com/api/v1/user/login', {
```
Comment (joshwambere) [inline src/views/Login.js]: use backend url from env

File: src/views/Login.js [hunk 6]
```diff
@@ -1,10 +1,107 @@
-import React from 'react'
+/* eslint-disable max-len */
+/* eslint-disable no-unused-vars */
```
Comment (joshwambere) [inline src/views/Login.js]: why did you disable this eslint rurle?

**Review by joshwambere — CHANGES_REQUESTED** [review]

**Review by uwambajeddy — APPROVED** [review]

**Review by Patricknshimiyimana — APPROVED** [review]

**Review by joshwambere — APPROVED** [review]

**Pull request merged**